src/main/psk-ui/src/api/model/create-team-request.ts

```diff
@@ -0,0 +1,5 @@
+export interface CreateTeamRequest {
+	name: string;
+	goal: string | null | undefined;
```

Comment:
shouldn't it be LearningTopic[]?

Reply:
We will have a different PR for team goals.